### PR TITLE
Support Neptune-ML Profile with Export-PG-From-Queries

### PR DESCRIPTION
**Pull request by Cole-Greer**
Issue #, if available: #44

Description of changes:

This PR updates the --structured-output mode of export-pg-from-queries to add support for the Neptune-ML profile. Includes output of stats.json, config.json, and training-data-configuration.json.

_Reminder: Add relevant entry to CHANGELOG.md_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazonaws/services/neptune/propertygraph/StatsContainer.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.amazonaws.services.neptune.propertygraph;
```
Comment (xiazcy) [inline src/main/java/com/amazonaws/services/neptune/propertygraph/StatsContainer.java:13]: New file is missing license header.

File: src/test/java/com/amazonaws/services/neptune/ExportServiceIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
+package com.amazonaws.services.neptune;
```
Comment (xiazcy) [inline src/test/java/com/amazonaws/services/neptune/ExportServiceIntegrationTest.java:13]: License header missing

File: src/test/java/com/amazonaws/services/neptune/profiles/neptune_ml/NeptuneMachineLearningExportEventHandlerV2Test.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
+package com.amazonaws.services.neptune.profiles.neptune_ml;
```
Comment (xiazcy) [inline src/test/java/com/amazonaws/services/neptune/profiles/neptune_ml/NeptuneMachineLearningExportEventHandlerV2Test.java:13]: License header.

File: src/test/java/com/amazonaws/services/neptune/profiles/neptune_ml/NeptuneMachineLearningExportEventHandlerV1Test.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
+package com.amazonaws.services.neptune.profiles.neptune_ml;
```
Comment (xiazcy) [inline src/test/java/com/amazonaws/services/neptune/profiles/neptune_ml/NeptuneMachineLearningExportEventHandlerV1Test.java:13]: License header

File: src/test/java/com/amazonaws/services/neptune/propertygraph/io/QueryTaskTest.java [hunk 5]
```diff
@@ -1,15 +1,23 @@
 package com.amazonaws.services.neptune.propertygraph.io;
```
Comment (xiazcy) [inline src/test/java/com/amazonaws/services/neptune/propertygraph/io/QueryTaskTest.java:13]: Missing license header

**Review by xiazcy — APPROVED** [review]
Some files are missing license headers, while I believe they are not completely necessary given we've got a license file at root, I'd add them to stay consistent with the rest of the package.
Also it will be nice to test our the ML export with the pipeline if you could get it set up, but I'm ok with doing the output comparison with normal ML export to make sure the format is correct here.

**Comment by Cole-Greer** [comment]
> Some files are missing license headers, while I believe they are not completely necessary given we've got a license file at root, I'd add them to stay consistent with the rest of the package. Also it will be nice to test our the ML export with the pipeline if you could get it set up, but I'm ok with doing the output comparison with normal ML export to make sure the format is correct here.

I have added the missing license headers, I will follow-up with a separate PR which adds the apache-rat-plugin to maven to automatically detect such errors in the future. I have verified that export-pg-from-queries with the ml profile produces identical output to export-pg with ml.

**Pull request merged**